.github/workflows/integration_tests.yml

```diff
@@ -88,3 +88,72 @@ jobs:
               echo 'Expected "1, 1" (alert groups, alerts). They were not created in 30 seconds during this integration test. Something is broken' >&2
               exit 1
           fi
+
+  ui-integration-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
a lot of this is copy/paste from the previous job.

Ideally we'd be able to create a CI workflow as such:
- deploy-helm-release
  - backend-interations-tests (dependent on parent job)
  - ui-interations-tests (dependent on parent job)

look into [sharing docker containers between jobs](https://github.com/docker/build-push-action/issues/225)